<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessSchedulerDialog</class>
 <widget class="QDialog" name="ProcessSchedulerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SJFButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SJF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="readyQueueDisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>104</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready Queue:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="completedDisplay">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>104</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Sim</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>106</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Completed Processes:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
